src/build.py

```diff
@@ -840,6 +858,27 @@ def V8():
     CopyBinaryToArchive(os.path.join(V8_OUT_DIR, a))
 
 
+def Jsc():
+  buildbot.Step('JSC')
+  Mkdir(JSC_OUT_DIR)
+
+  proc.check_call(['xcrun', PREBUILT_CMAKE_BIN, '-Wno-dev',
```

Comment:
This might potentially be a problem because of the way that dev tools are "installed" on Chromium infrastructure; i.e. they are not really installed per se on the system, so sometimes assumptions about the installation are wrong, etc. Practically it may just be that we need an explicit path to xcrun or to add something to the $PATH or something. I'm trying it out now, will let you know